<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="mapView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>391</width>
      <height>351</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>570</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>26</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>U</string>
    </property>
   </widget>
   <widget class="QToolButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>26</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QToolButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>26</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QToolButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>530</y>
      <width>26</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensor1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor1data">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor1data</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor2">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>180</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensor2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor2data">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor2data</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensor3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor3data">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor3data</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor4:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor4data">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor4data</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor5">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>530</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor5:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor5data">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>550</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor5data</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor6data">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>670</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor6data</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensor6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>650</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor6:</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>600</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedPercent">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>600</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="sensorPlot1" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QCustomPlot" name="sensorPlot2" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QCustomPlot" name="sensorPlot3" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QCustomPlot" name="sensorPlot4" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QCustomPlot" name="sensorPlot5" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QCustomPlot" name="sensorPlot6" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>630</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connecting...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>387</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QToolButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>26</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fetchButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Fetch</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionForward"/>
    <addaction name="actionDown"/>
    <addaction name="actionLeft"/>
    <addaction name="actionRight"/>
    <addaction name="actionSpeedUp"/>
    <addaction name="actionSlowDown"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuCommands"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="text">
    <string>Backward</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Left</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionSpeedUp">
   <property name="text">
    <string>SpeedUp</string>
   </property>
   <property name="shortcut">
    <string>Alt+Up</string>
   </property>
  </action>
  <action name="actionSlowDown">
   <property name="text">
    <string>SlowDown</string>
   </property>
   <property name="shortcut">
    <string>Alt+Down</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
